config/routes.rb

```diff
@@ -1,13 +1,13 @@
 Rails.application.routes.draw do
   root "users#feed"
-
+  
   devise_for :users
   
   resources :comments
   resources :follow_requests, except: [:index, :show, :new, :edit]
   resources :likes, only: [:create, :destroy]
   resources :photos, except: [:index]
-
+  
```

Comment:
remove extra spaces to keep a clean diff

Reply:
I'm not sure how to fix this. It is the same line? Line 10 but was added for spacing between resources.